src/activity_type.h

```diff
@@ -3,10 +3,13 @@
 #define CATA_SRC_ACTIVITY_TYPE_H
 
 #include <string>
+#include <optional>
 
 #include "catalua_type_operators.h"
 #include "string_id.h"
 #include "translations.h"
+#include <character_stat.h>
```

Comment:
```suggestion
#include "character_stat.h"
```
character_stat.h is a project include not system include